test/jb_token/transfer_ownership.test.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { deployJbToken } from '../helpers/utils';
+
+describe('JBToken::transferOwnership(...)', function () {
+  const name = 'TestTokenDAO';
+  const symbol = 'TEST';
+
+  async function setup() {
+    const [deployer, ...addrs] = await ethers.getSigners();
+    const testToken = await deployJbToken(name, symbol);
+    return { deployer, addrs, testToken };
+  }
+
+  it('Should transfer ownership to another address if caller is owner', async function () {
+    const { deployer, addrs, testToken } = await setup();
+    const newAddr = addrs[0];
+
+    const transferOwnershipTx = await testToken
+      .connect(deployer)
+      .transferOwnership(newAddr.address);
+
+    await expect(transferOwnershipTx)
+      .to.emit(testToken, 'OwnershipTransferred')
+      .withArgs(deployer.address, newAddr.address);
+
+    const balance = await testToken.owner();
```

Comment:
this should be 'owner', right? Also consider just throwing it into the expect

expect(await testToken.owner()).to.equal(newAddr.address)

If the logic inside the expect is pretty terse this is generally what I do – consider replacing other instances.

Reply:
Oh yeah good catch